### ISSUE TITLE
standby master node gone with exeption:the limit of 250 distributed transactions has been reached (cdbdtxrecovery.c:569)

### ISSUE THREAD

**Issue by tangyibo** [greenplum-db/gpdb#10779]
### Greenplum version or build
```
postgres (Greenplum Database) 6.6.0 build commit:12ef7352da41898153e3c7049c5f5c32841239d4
```
### OS version and uname -a

```
[gpadmin@master1 ~]$ uname -a
Linux smdw 3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```
### Actual behavior

standby master node gone with exeption, log information is 

```
2020-09-04 01:32:07.963567 CST,,,p9852,th1468024960,,,,0,,,seg-1,,,,,"FATAL","XX000","the limit of 250 distributed transactions has been reached (cdbdtxrecovery.c:569)","The global user configuration (GUC) server parameter max_prepared_transactions controls this limit.",,,,"xlog redo distributed commit 2020-09-04 01:32:07.868976+08 gid = 3745140736-0000032766 gid = 1599014694-0008353353, gxid = 8353353",,0,,"cdbdtxrecovery.c",569,"Stack trace:
1    0x9fa23c postgres errstart (elog.c:557)
2    0xae24a3 postgres redoDistributedCommitRecord (cdbdtxrecovery.c:565)
3    0x54309f postgres xact_redo (xact.c:6324)
4    0x5590f2 postgres StartupXLOG (xlog.c:7283)
5    0x81ca83 postgres StartupProcessMain (startup.c:248)
6    0x596042 postgres AuxiliaryProcessMain (bootstrap.c:447)
7    0x818b5c postgres <symbol not found> (postmaster.c:5817)
8    0x81c27e postgres PostmasterMain (postmaster.c:1502)
9    0x4cf2a1 postgres main (main.c:205)
10   0x7f1353163555 libc.so.6 __libc_start_main + 0xf5
11   0x4cf87c postgres <symbol not found> + 0x4cf87c
"
2020-09-04 01:32:07.977114 CST,,,p9825,th1468024960,,,,0,,,seg-1,,,,,"LOG","00000","startup process (PID 9852) exited with exit code 1",,,,,,,0,,"postmaster.c",3933,
2020-09-04 01:32:07.977187 CST,,,p9825,th1468024960,,,,0,,,seg-1,,,,,"LOG","00000","terminating any other active server processes",,,,,,,0,,"postmaster.c",3672,
```

I want to know how to avoid this exeption? Thanks!

**Comment by paul-guo-** [comment]
I think this was fixed in 0c57d9fc75c4a4498e3cbdeec5174dc22a5b216b, which was contained in 6.10.0 and above. Please upgrade and try again.

**Comment by tangyibo** [comment]
Thank you very much! I will do it now!
